src/test/test_pidfile.sh

```diff
@@ -71,7 +71,7 @@ function TEST_pidfile() {
     # another daemon
     cp $dir/osd.0.pid $dir/osd.0.pid.old  # so that kill_daemon finds the pid
```

Comment:
Good catch ! I think it is better to rename osd.0.old.pid so that kill_daemons finds it. And also in other places where the script expects that the file is found by kill_daemons.